### PR TITLE
[KOA-5084] BPKStarRating SwiftUI

### PR DESCRIPTION
**Pull request by frugoman**
|Day|Night|
|---|---|
|iPhone-swiftui_star-rating___all_lm|iPhone-swiftui_star-rating___all_dm|

### CODE REVIEW

**Review by marianeum — COMMENTED** [review]
couple of minor comments

File: Backpack-SwiftUI/StarRating/Classes/BPKHotelStarRating.swift [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+/// A view that displays a star rating based on a rating value.
+public struct BPKHotelStarRating: View {
+    @Binding private var rating: Float
+    private let maxRating: Int
+    private let size: BPKStarRatingSize
+    
+    /// Initializes a new `BPKHotelStarRating` instance.
+    /// - Parameters:
+    ///   - rating: A binding to the current rating value.
+    ///   - maxRating: The maximum rating value allowed.
```
[the remark this answers is not in the document]
Comment (frugoman) [reply Backpack-SwiftUI/StarRating/Classes/BPKHotelStarRating.swift:30]: ah right, it doesnt really make sense to have this in the Hotel Star rating.. I'll remove it!

File: Backpack-SwiftUI/StarRating/Classes/BPKHotelStarRating.swift [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    ///   Represents the number of stars to display.
+    ///   - size: The size of the star rating, defaults to `.small`.
+    public init(
+        rating: Binding<Float>,
```
[the remark this answers is not in the document]
Comment (frugoman) [reply Backpack-SwiftUI/StarRating/Classes/BPKHotelStarRating.swift:34]: yeoppp

File: Backpack-SwiftUI/StarRating/README.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+[![Cocoapods](https://img.shields.io/cocoapods/v/Backpack-SwiftUI.svg?style=flat)](hhttps://cocoapods.org/pods/Backpack-SwiftUI)
+[![class reference](https://img.shields.io/badge/Class%20reference-iOS-blue)](https://backpack.github.io/ios/versions/latest/swiftui/Structs/BPKStarRating.html)
+[![view on Github](https://img.shields.io/badge/Source%20code-GitHub-lightgrey)](https://github.com/Skyscanner/backpack-ios/tree/main/Backpack-SwiftUI/StarRating)
+
+| Day | Night |
+| --- | --- |
+| <img src="https://raw.githubusercontent.com/Skyscanner/backpack-ios/main/screenshots/iPhone-swiftui_star-rating___all_lm.png" alt="" width="375" /> |<img src="https://raw.githubusercontent.com/Skyscanner/backpack-ios/main/screenshots/iPhone-swiftui_star-rating___all_dm.png" alt="" width="375" /> |
+
+## Usage
```
Comment (marianeum) [inline Backpack-SwiftUI/StarRating/README.md:11]: can you also add the hotels version here please?

**Review by marianeum — APPROVED** [review]

**Pull request merged**